src/components/SummaryCard/SummaryCardContainer/__tests__/SummaryCardContainer.spec.js

```diff
@@ -61,39 +139,171 @@ describe('SummaryCardContainer', () => {
       />
     );
 
-    const { getByTestId } = container;
+    userEvent.tab();
+
+    // The first summary card select:
+    expect(getByLabelText(/test select 0/i)).toHaveFocus();
+
+    userEvent.tab();
 
-    batchActions = getByTestId(batchActionsId);
-    select = getByTestId(`${selectId}${summaryCards[0].id}`);
+    // Loop complete.
+    // The first summary card select:
+    expect(getByLabelText(/test select 1/i)).toHaveFocus();
+
+    userEvent.tab();
```

Comment:
Would you mind elaborating on the purpose of this last tab?

Reply:
No purpose for it -- looks like it was left there by mistake.